src/arch/xtensa/CMakeLists.txt

```diff
@@ -301,16 +301,14 @@ if(MEU_PATH)
 		set(MEU_OFFSET 1088)
 	endif()
 
-	set(otc_private_key ${PROJECT_SOURCE_DIR}/rimage/keys/otc_private_key.pem)
```

Comment:
What about people that dont use scripts?
We need to set default like:
```
if(NOT DEFINED RIMAGE_PRIVATE_KEY)
	set(RIMAGE_PRIVATE_KEY ${PROJECT_SOURCE_DIR}/rimage/keys/otc_private_key.pem)
endif()
```

Reply:
ok. I know next to nothing on Cmake syntax so taking out the whole thing was simpler :-)